openapi/propolis-server-falcon.json

```diff
@@ -112,6 +112,41 @@
         }
       }
     },
+    "/instance/disk/{id}/status": {
```

Comment:
The only question I have is if this propolis-server-falcon needs any plumbing
to pass this to crucible if it needs to.  Given the other endpoints must already
be doing the same thing, it's probably okay.